[ qtdeclarative[core]] Cannot open include file: 'qquicksaturationlightnesspicker_p.h': No such file or directory

**Host Environment**

- OS: Windows11
- Compiler: msvc vc413

**To Reproduce**

Steps to reproduce the behavior:

`.\vcpkg install pcl[apps,cuda,opengl,openni2,pcap,qt,simulation,tools,visualization,vtk]:x64-windows ` or
`.\vcpkg install qt:x64-windows `

**Failure logs**

-- Building x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: D:/ProgramFiles/CMake/bin/cmake.exe --build . --config Debug --target install -- -v -j13
    Working Directory: D:/ProgramFiles/vcpkg/buildtrees/qtdeclarative/x64-windows-dbg
    See logs for more information:
      D:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\install-x64-windows-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  installed/x64-windows/share/qtbase/qt_install_submodule.cmake:337 (vcpkg_cmake_install)
  ports/qtdeclarative/portfile.cmake:22 (qt_install_submodule)
  scripts/ports.cmake:147 (include)
**Additional context**
[2843/2933] D:\ProgramFiles\Microsoft\VisualStudio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe   /TP -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_QUICKDIALOGS2QUICKIMPL_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DEPRECATED_WARNINGS_SINCE=0x070000 -DQT_DISABLE_DEPRECATED_BEFORE=0x040800 -DQT_GUI_LIB -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NETWORK_LIB -DQT_NO_CAST_FROM_ASCII -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_OPENGL_LIB -DQT_QMLINTEGRATION_LIB -DQT_QMLMODELS_LIB -DQT_QML_LIB -DQT_QUICKCONTROLS2IMPL_LIB -DQT_QUICKDIALOGS2UTILS_LIB -DQT_QUICKTEMPLATES2_LIB -DQT_QUICK_LIB -DQT_USE_QSTRINGBUILDER -DQuickDialogs2QuickImpl_EXPORTS -DUNICODE -DWIN32 -DWIN64 -D_CRT_SECURE_NO_WARNINGS -D_ENABLE_EXTENDED_ALIGNED_STORAGE -D_UNICODE -D_USE_MATH_DEFINES -D_WIN64 -D_WINDLL -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\quickdialogs2\quickdialogs2quickimpl\QuickDialogs2QuickImpl_autogen\include -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickDialogs2QuickImpl -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\src\here-src-6-a31aea0fcd.clean\src\quickdialogs2\quickdialogs2quickimpl -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\quickdialogs2\quickdialogs2quickimpl -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickDialogs2QuickImpl\6.4.2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickDialogs2QuickImpl\6.4.2\QtQuickDialogs2QuickImpl -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\qml -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQml\6.4.2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQml\6.4.2\QtQml -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQml -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQmlIntegration -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\src\here-src-6-a31aea0fcd.clean\src\qmlintegration -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\qmlintegration -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuick -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQmlModels -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\quick -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuick\6.4.2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuick\6.4.2\QtQuick -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\qmlmodels -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQmlModels\6.4.2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQmlModels\6.4.2\QtQmlModels -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickTemplates2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\quicktemplates2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickTemplates2\6.4.2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickTemplates2\6.4.2\QtQuickTemplates2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\quickcontrols2impl -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickControls2Impl\6.4.2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickControls2Impl\6.4.2\QtQuickControls2Impl -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickControls2Impl -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickDialogs2Utils -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\quickdialogs2\quickdialogs2utils -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickDialogs2Utils\6.4.2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickDialogs2Utils\6.4.2\QtQuickDialogs2Utils -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtCore\6.4.2 -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtCore\6.4.2\QtCore -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtCore -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6 -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\share\Qt6\mkspecs\win32-msvc -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtNetwork -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtGui -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtOpenGL -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtGui\6.4.2 -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtGui\6.4.2\QtGui -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /W3 /EHs-c- /wd4530 /wd4577 -Zc:__cplusplus -permissive- -utf-8 -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:externConstexpr -Zc:wchar_t -bigobj -std:c++17 /bigobj /YuD:/ProgramFiles/vcpkg/buildtrees/qtdeclarative/x64-windows-dbg/src/quickdialogs2/quickdialogs2quickimpl/CMakeFiles/QuickDialogs2QuickImpl.dir/cmake_pch.hxx /FpD:/ProgramFiles/vcpkg/buildtrees/qtdeclarative/x64-windows-dbg/src/quickdialogs2/quickdialogs2quickimpl/CMakeFiles/QuickDialogs2QuickImpl.dir/./cmake_pch.cxx.pch /FID:/ProgramFiles/vcpkg/buildtrees/qtdeclarative/x64-windows-dbg/src/quickdialogs2/quickdialogs2quickimpl/CMakeFiles/QuickDialogs2QuickImpl.dir/cmake_pch.hxx /showIncludes /Fosrc\quickdialogs2\quickdialogs2quickimpl\CMakeFiles\QuickDialogs2QuickImpl.dir\quickdialogs2quickimpl_qmltyperegistrations.cpp.obj /Fdsrc\quickdialogs2\quickdialogs2quickimpl\CMakeFiles\QuickDialogs2QuickImpl.dir\ /FS -c D:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\quickdialogs2\quickdialogs2quickimpl\quickdialogs2quickimpl_qmltyperegistrations.cpp
FAILED: src/quickdialogs2/quickdialogs2quickimpl/CMakeFiles/QuickDialogs2QuickImpl.dir/quickdialogs2quickimpl_qmltyperegistrations.cpp.obj 
D:\ProgramFiles\Microsoft\VisualStudio\2022\Community\VC\Tools\MSVC\14.34.31933\bin\Hostx64\x64\cl.exe   /TP -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_QUICKDIALOGS2QUICKIMPL_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DEPRECATED_WARNINGS_SINCE=0x070000 -DQT_DISABLE_DEPRECATED_BEFORE=0x040800 -DQT_GUI_LIB -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NETWORK_LIB -DQT_NO_CAST_FROM_ASCII -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_OPENGL_LIB -DQT_QMLINTEGRATION_LIB -DQT_QMLMODELS_LIB -DQT_QML_LIB -DQT_QUICKCONTROLS2IMPL_LIB -DQT_QUICKDIALOGS2UTILS_LIB -DQT_QUICKTEMPLATES2_LIB -DQT_QUICK_LIB -DQT_USE_QSTRINGBUILDER -DQuickDialogs2QuickImpl_EXPORTS -DUNICODE -DWIN32 -DWIN64 -D_CRT_SECURE_NO_WARNINGS -D_ENABLE_EXTENDED_ALIGNED_STORAGE -D_UNICODE -D_USE_MATH_DEFINES -D_WIN64 -D_WINDLL -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\quickdialogs2\quickdialogs2quickimpl\QuickDialogs2QuickImpl_autogen\include -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickDialogs2QuickImpl -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\src\here-src-6-a31aea0fcd.clean\src\quickdialogs2\quickdialogs2quickimpl -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\quickdialogs2\quickdialogs2quickimpl -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickDialogs2QuickImpl\6.4.2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickDialogs2QuickImpl\6.4.2\QtQuickDialogs2QuickImpl -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\qml -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQml\6.4.2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQml\6.4.2\QtQml -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQml -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQmlIntegration -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\src\here-src-6-a31aea0fcd.clean\src\qmlintegration -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\qmlintegration -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuick -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQmlModels -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\quick -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuick\6.4.2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuick\6.4.2\QtQuick -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\qmlmodels -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQmlModels\6.4.2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQmlModels\6.4.2\QtQmlModels -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickTemplates2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\quicktemplates2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickTemplates2\6.4.2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickTemplates2\6.4.2\QtQuickTemplates2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\quickcontrols2impl -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickControls2Impl\6.4.2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickControls2Impl\6.4.2\QtQuickControls2Impl -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickControls2Impl -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickDialogs2Utils -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\quickdialogs2\quickdialogs2utils -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickDialogs2Utils\6.4.2 -ID:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickDialogs2Utils\6.4.2\QtQuickDialogs2Utils -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtCore\6.4.2 -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtCore\6.4.2\QtCore -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtCore -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6 -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\share\Qt6\mkspecs\win32-msvc -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtNetwork -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtGui -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtOpenGL -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtGui\6.4.2 -external:ID:\ProgramFiles\vcpkg\installed\x64-windows\include\Qt6\QtGui\6.4.2\QtGui -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MDd /W3 /EHs-c- /wd4530 /wd4577 -Zc:__cplusplus -permissive- -utf-8 -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:externConstexpr -Zc:wchar_t -bigobj -std:c++17 /bigobj /YuD:/ProgramFiles/vcpkg/buildtrees/qtdeclarative/x64-windows-dbg/src/quickdialogs2/quickdialogs2quickimpl/CMakeFiles/QuickDialogs2QuickImpl.dir/cmake_pch.hxx /FpD:/ProgramFiles/vcpkg/buildtrees/qtdeclarative/x64-windows-dbg/src/quickdialogs2/quickdialogs2quickimpl/CMakeFiles/QuickDialogs2QuickImpl.dir/./cmake_pch.cxx.pch /FID:/ProgramFiles/vcpkg/buildtrees/qtdeclarative/x64-windows-dbg/src/quickdialogs2/quickdialogs2quickimpl/CMakeFiles/QuickDialogs2QuickImpl.dir/cmake_pch.hxx /showIncludes /Fosrc\quickdialogs2\quickdialogs2quickimpl\CMakeFiles\QuickDialogs2QuickImpl.dir\quickdialogs2quickimpl_qmltyperegistrations.cpp.obj /Fdsrc\quickdialogs2\quickdialogs2quickimpl\CMakeFiles\QuickDialogs2QuickImpl.dir\ /FS -c D:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\src\quickdialogs2\quickdialogs2quickimpl\quickdialogs2quickimpl_qmltyperegistrations.cpp
cl : Command line warning D9025 : overriding '/EHs' with '/EHs-'
cl : Command line warning D9025 : overriding '/EHc' with '/EHc-'
D:\ProgramFiles\vcpkg\buildtrees\qtdeclarative\x64-windows-dbg\include\QtQuickDialogs2QuickImpl\6.4.2\QtQuickDialogs2QuickImpl\private/qquicksaturationlightnesspicker_p.h(1): fatal error C1083: Cannot open include file: '../../../../../../src/here-src-6-a31aea0fcd.clean/src/quickdialogs2/quickdialogs2quickimpl/qquicksaturationlightnesspicker_p.h': No such file or directory

--- a/ports/qtdeclarative/portfile.cmake
+++ b/ports/qtdeclarative/portfile.cmake
@@ -3,6 +3,7 @@ include("${SCRIPT_PATH}/qt_install_submodule.cmake")
 
 string(LENGTH "${CURRENT_BUILDTREES_DIR}" buildtrees_path_length)
 if(buildtrees_path_length GREATER 44 AND CMAKE_HOST_WIN32)
+    vcpkg_buildpath_length_warning(44)
     message(WARNING "The ${PORT} source was will be extracted to ${CURRENT_BUILDTREES_DIR} , which has more then 44 characters in length.")
     message(FATAL_ERROR "terminating due to ${CURRENT_BUILDTREES_DIR} being too long. Pass --x-buildtrees-root=<shortpath> to vcpkg")
 endif()

--- a/ports/qtdeclarative/portfile.cmake
+++ b/ports/qtdeclarative/portfile.cmake
@@ -1,12 +1,7 @@
 set(SCRIPT_PATH "${CURRENT_INSTALLED_DIR}/share/qtbase")
 include("${SCRIPT_PATH}/qt_install_submodule.cmake")
 
-string(LENGTH "${CURRENT_BUILDTREES_DIR}" buildtrees_path_length)
-if(buildtrees_path_length GREATER 44 AND CMAKE_HOST_WIN32)
-    vcpkg_buildpath_length_warning(44)
-    message(WARNING "The ${PORT} source was will be extracted to ${CURRENT_BUILDTREES_DIR} , which has more then 44 characters in length.")
-    message(FATAL_ERROR "terminating due to ${CURRENT_BUILDTREES_DIR} being too long. Pass --x-buildtrees-root=<shortpath> to vcpkg")
-endif()
+vcpkg_buildpath_length_warning(44)
 
  set(TOOL_NAMES 
         qml

--- a/ports/qtdeclarative/portfile.cmake
+++ b/ports/qtdeclarative/portfile.cmake
@@ -1,6 +1,13 @@
 set(SCRIPT_PATH "${CURRENT_INSTALLED_DIR}/share/qtbase")
 include("${SCRIPT_PATH}/qt_install_submodule.cmake")
 
+string(LENGTH "${CURRENT_BUILDTREES_DIR}" buildtrees_path_length)
+if(buildtrees_path_length GREATER 44 AND CMAKE_HOST_WIN32)
+    vcpkg_buildpath_length_warning(44)
+    message(WARNING "The ${PORT} source was will be extracted to ${CURRENT_BUILDTREES_DIR} , which has more then 44 characters in length.")
+    message(FATAL_ERROR "terminating due to ${CURRENT_BUILDTREES_DIR} being too long.")
+endif()
+
  set(TOOL_NAMES 
         qml
         qmlcachegen

--- a/ports/qtdeclarative/portfile.cmake
+++ b/ports/qtdeclarative/portfile.cmake
@@ -3,9 +3,8 @@ include("${SCRIPT_PATH}/qt_install_submodule.cmake")
 
 string(LENGTH "${CURRENT_BUILDTREES_DIR}" buildtrees_path_length)
 if(buildtrees_path_length GREATER 44 AND CMAKE_HOST_WIN32)
-    vcpkg_buildpath_length_warning(44)
     message(WARNING "The ${PORT} source was will be extracted to ${CURRENT_BUILDTREES_DIR} , which has more then 44 characters in length.")
-    message(FATAL_ERROR "terminating due to ${CURRENT_BUILDTREES_DIR} being too long.")
+    message(FATAL_ERROR "terminating due to ${CURRENT_BUILDTREES_DIR} being too long. Pass --x-buildtrees-root=<shortpath> to vcpkg")
 endif()
 
  set(TOOL_NAMES 

--- a/ports/qtdeclarative/vcpkg.json
+++ b/ports/qtdeclarative/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "qtdeclarative",
   "version": "6.4.2",
+  "port-version": 1,
   "description": "Qt Declarative (Quick 2)",
   "homepage": "https://www.qt.io/",
   "license": null,

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6418,7 +6418,7 @@
     },
     "qtdeclarative": {
       "baseline": "6.4.2",
-      "port-version": 0
+      "port-version": 1
     },
     "qtdeviceutilities": {
       "baseline": "6.4.2",

--- a/versions/q-/qtdeclarative.json
+++ b/versions/q-/qtdeclarative.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "9eee650bf7c4489a3ae324dd3a0d6ba85db65057",
+      "git-tree": "3f5d46fcaae15b0c653996603f73f91954e4f0be",
       "version": "6.4.2",
       "port-version": 1
     },

--- a/versions/q-/qtdeclarative.json
+++ b/versions/q-/qtdeclarative.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "6c0fa965ab59fbd00e83c2b1f328e9f4ec4abb46",
+      "git-tree": "70badba61897e63a26540f70ff53e0df92ff01bb",
       "version": "6.4.2",
       "port-version": 1
     },

--- a/versions/q-/qtdeclarative.json
+++ b/versions/q-/qtdeclarative.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "9eee650bf7c4489a3ae324dd3a0d6ba85db65057",
+      "version": "6.4.2",
+      "port-version": 1
+    },
+    {
       "git-tree": "005e51361039c1a03018a94d3f2da31bea63ab59",
       "version": "6.4.2",
       "port-version": 0

--- a/versions/q-/qtdeclarative.json
+++ b/versions/q-/qtdeclarative.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "3f5d46fcaae15b0c653996603f73f91954e4f0be",
+      "git-tree": "6c0fa965ab59fbd00e83c2b1f328e9f4ec4abb46",
       "version": "6.4.2",
       "port-version": 1
     },

